### PR TITLE
feat: community spend pool

### PR DESCRIPTION
**Pull request by ivan-angjelkoski**
### In This PR:
- Added fetchCommunitySpendPool transactions

### CODE REVIEW

File: packages/sdk-ts/src/client/indexer/types/explorer-rest.ts [hunk 1]
```diff
@@ -225,3 +225,19 @@ export interface CW20BalanceExplorerApiResponse {
...
+
+export interface BankTransferFromExplorerApiResponse {
+  sender: string
+  recipient: string
+  amounts: { denom: string; amount: string }[]
+  block_number: number
+  block_timestamp: string
+}
+
+export interface BankTransfer {
```
Comment (shane-moore) [inline packages/sdk-ts/src/client/indexer/types/explorer-rest.ts:237]: can you move `BankTransfer` to `sdk-ts/src/client/indexer/types/explorer.ts`?

also, let's use `import { Coin } from '@injectivelabs/ts-types'` to replace the `{ denom: string; amount: string }`

File: packages/sdk-ts/src/client/indexer/types/explorer-rest.ts [hunk 2]
```diff
@@ -225,3 +225,19 @@ export interface CW20BalanceExplorerApiResponse {
 }
 
 export { BaseTransaction }
+
+export interface BankTransferFromExplorerApiResponse {
+  sender: string
+  recipient: string
+  amounts: { denom: string; amount: string }[]
```
Comment (shane-moore) [inline packages/sdk-ts/src/client/indexer/types/explorer-rest.ts:232]: let's use `import { Coin } from '@injectivelabs/ts-types'` to replace the `{ denom: string; amount: string }`

**Review by shane-moore — CHANGES_REQUESTED** [review]

File: packages/sdk-ts/src/client/indexer/rest/IndexerRestExplorerApi.ts [hunk 3]
```diff
@@ -627,4 +629,55 @@ export class IndexerRestExplorerApi extends BaseRestConsumer {
       })
     }
   }
+
+  async fetchCommunitySpendPool(params: {
```
Comment (bangjelkoski) [inline packages/sdk-ts/src/client/indexer/rest/IndexerRestExplorerApi.ts:633]: Nope, we should have `fetchBankTransfers` and then pass the `is_community_pool_related` as a parameter on the function call.

**Review by bangjelkoski — CHANGES_REQUESTED** [review]

File: packages/sdk-ts/src/client/indexer/rest/IndexerRestExplorerApi.ts (hunk 3 above)
Comment (bangjelkoski) [reply packages/sdk-ts/src/client/indexer/rest/IndexerRestExplorerApi.ts]: I've resolved it.

**Pull request merged**